contracts/CurveCalcMeta.vy

```diff
@@ -39,7 +39,7 @@ interface CurveBase:
 
 
 MAX_COINS: constant(int128) = 8
-INPUT_SIZE: constant(int128) = 100
```

Comment:
Changed `INPUT_SIZE` to `80` as the contract size was exceeding the bytecode size limit. Was there a particular reason for setting it to `100`?

Reply:
Just some number. Doesn't matter how big it is I think as long as it fits, but should be big enough. That's for DEX aggregators to do this on-chain (where INPUT_SIZE is the maximum they do)